ci/tests/puppeteer/scenarios/passwordless-login-with-reset-password/script.json

```diff
@@ -0,0 +1,33 @@
+{
+  "dependencies": "passwordless-webflow,pm-webflow",
```

Comment:
pm-webflow is included by default. I don't think you need to forcefully let it live here.

Reply:
Indeed, this is not needed. The test passes without `pm-webflow`.

I will remove it.